index.html

```diff
@@ -6853,13 +6858,15 @@ <h4>JsonLdErrorCode</h4>
     </li>
```

Comment:
Just to answer a question that the Director may have, let us add a sentence of the sort "All these changes are editorial and do not affect the observable behavior of the API" (provided this statement is correct:-)

Reply:
Done.